docs/data/joy/getting-started/templates/sign-in-side/App.tsx

```diff
@@ -154,58 +146,79 @@ export default function JoySignInSideTemplate() {
               },
             }}
           >
-            <div>
-              <Typography component="h1" fontSize="xl2" fontWeight="lg">
-                Sign in to your account
-              </Typography>
```

Comment:
We removed the `component="h1"` but didn't add it back. It's best for SEO for the developers who will use this. https://app.ahrefs.com/site-audit/3524616/106/data-explorer?columns=pageRating%2Curl%2Ctraffic%2ChttpCode%2Cdepth%2Ch1%2Ch1Length%2CnrH1%2Ccompliant&filterId=1337ec32524c0ae0a368c1dc8c7031c8&issueId=c64d9d0b-d0f4-11e7-8ed1-001e67ed4656. 

Fixed in 8770d34932b85d943b0f54582a3508c5256228d3